src/markdown/highlight/common.ts

```diff
@@ -4,6 +4,34 @@ export const sortHighlights = (a, b) => {
   return a.start > b.start ? 1 : -1
 };
 
+export const mergingHighlights = (highlights) => {
```

Comment:
I'm not totally sure that MMD should be handling 'merging' highlights though.

Reply:
I've just changed mergingHighlights function
Now, adjacent highlights will not be merged.

Reply:
Thanks @OlgaRedozubova!